mod.ts

```diff
@@ -57,6 +57,17 @@ export function serve(userRoutes: Routes): void {
   });
 }
 
+async function newResponse(res: Response, headers: Record<string, string>): Promise<Response> {
+    return new Response(await res.blob(), {
```

Comment:
```suggestion
    return new Response(await res.body, {
```

`blob()` might not work for streaming responses.

Reply:
Thanks for the correction!